Source/JavaScriptCore/inspector/agents/InspectorDebuggerAgent.cpp

```diff
@@ -965,20 +965,20 @@ Protocol::ErrorStringOr<void> InspectorDebuggerAgent::continueToLocation(Ref<JSO
 
 Protocol::ErrorStringOr<Ref<JSON::ArrayOf<Protocol::GenericTypes::SearchMatch>>> InspectorDebuggerAgent::searchInContent(const Protocol::Debugger::ScriptId& scriptId, const String& query, std::optional<bool>&& caseSensitive, std::optional<bool>&& isRegex)
 {
-    auto it = m_scripts.find(parseIntegerAllowingTrailingJunk<JSC::SourceID>(scriptId).value_or(0));
-    if (it == m_scripts.end())
+    auto scriptIterator = m_scripts.find(parseIntegerAllowingTrailingJunk<JSC::SourceID>(scriptId).value_or(0));
+    if (scriptIterator == m_scripts.end())
         return makeUnexpected("Missing script for given scriptId"_s);
 
     return ContentSearchUtilities::searchInTextByLines(it->value.source, query, caseSensitive.value_or(false), isRegex.value_or(false));
```

Comment:
You need to change `it` here as well.

Reply:
I changed `scriptIterator` to `injectedScript` to be consistent with the other injectedScripts.